### PR TITLE
Add basic ui.

### PR DESCRIPTION
**Pull request by qiuyesuifeng**
@c4pt0r  @siddontang @shenli  PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
PTAL @shenli

**Comment by shenli** [comment]
Why put UI in PD? I think it should be in TiAdmin.

**Comment by qiuyesuifeng** [comment]
@shenli  It is only a simple UI to let developers to have a try:)

**Comment by shenli** [comment]
LGTM
But I strongly recommend moving this to TiAdmin.

**Comment by qiuyesuifeng** [comment]
Agreed.
There will be a new UI in TiAdmin, this is only for simple use.
They are different.

File: server/api/router.go [hunk 1]
```diff
@@ -14,14 +14,19 @@
 package api
```
Comment (c4pt0r) [inline server/api/router.go]: license.

Comment (qiuyesuifeng) [inline server/api/router.go]: It is folded.

File: templates/static/js/app.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    "bg-black",
+    "bg-silver",
+    "bg-gray"
+];
+
+const eventRowTmpl = `
+<div class="jumbotron" ng-model="event">
+    <div class="jumbotron-contents">
+        <div class="row">
+            <div class="col-sm-1 log-img">
```
Comment (c4pt0r) [inline templates/static/js/app.js]: these code is ugly, I'm refactoring now.

**Comment by c4pt0r** [comment]
LGTM

**Pull request merged**